server/container_list.go

```diff
@@ -67,13 +69,17 @@ func (s *Server) ListContainers(ctx context.Context, req *pb.ListContainersReque
 
 	for _, ctr := range ctrList {
 		if err := s.runtime.UpdateStatus(ctr); err != nil {
-			return nil, err
+			logrus.Warnf("error updating status for ctr %s: %v", ctr.ID, err)
```

Comment:
If we receive an error here, do we really want to try processing this container?  Should we have a continue statement after this logrus.Warnf, drop processing on the current container and try the next one?  Your call, thought I'd ask the question.